packages/engine-ui/src/components/RouteActivities.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { FlyToInterpolator } from 'react-map-gl'
+import styled from 'styled-components'
+import { Activity, Route } from '../types'
+import helpers from '../utils/helpers'
+import { UIStateContext } from '../utils/UIStateContext'
+import Elements from './Elements'
+
+const ActivityGroup = styled.div`
+  margin-left: auto;
+  display: flex;
+  flex-direction: column;
```

Comment:
I think it's a bit more clear with some added space between the upphämtning and avlämning.
```suggestion
  flex-direction: column;
  margin-bottom: 2rem;
```